.github/workflows/unittests.yml

```diff
@@ -30,6 +30,10 @@
          with:
            environment-file: ${{ matrix.environment-file }}
            micromamba-version: 'latest'
+      
+       - name: install pulp via pip
```

Comment:
let's go ahead move `pulp` into the `.ci/*` environments and also `requirements.txt` & `environment.yml`

Reply:
Seems that if I run tests installing `pulp` from `micromamba` it can't run correctly because it doesn't install `PULP_CBC_CMD`. All locate tests use this solver.

Reply:
> Seems that if I run tests installing `pulp` from `micromamba` it can't run correctly because it doesn't install `PULP_CBC_CMD`. All locate tests use this solver.

@knaaptime Do you think [`cibuildwheel`](https://github.com/pypa/cibuildwheel) would be able to help with this issue?

Reply:
i think this might be a good excuse for us to check out [pyproject.toml](https://snarky.ca/what-the-heck-is-pyproject-toml/) files, which i'm learning are designed to help solve issues like this. Might also think about [poetry](https://python-poetry.org/docs/), though im not sold on that yet

Reply:
> i think this might be a good excuse for us to check out `pyproject.toml` files, which i'm learning are designed to help solve issues like this. Might also think about `poetry`, though im not sold on that yet

I'd be down to try the [`pyproject.toml`](https://snarky.ca/what-the-heck-is-pyproject-toml/) approach. Would you be willing to put together a PR?